api/src/main/java/io/strimzi/api/kafka/model/balancing/CruiseControlBrokerCapacity.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.balancing;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.UnknownPropertyPreserving;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Pattern;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Representation of the Cruise Control broker capacity settings. Since the Kafka brokers
+ * in Strimzi are homogeneous, the capacity values for each resource will be
+ * used for every broker.
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"disk", "cpu", "nwIn", "nwOut"})
+@EqualsAndHashCode
+public class CruiseControlBrokerCapacity implements UnknownPropertyPreserving, Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private Integer disk;
+    private Integer cpu;
+    private Integer nwIn;
+    private Integer nwOut;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    @Description("Broker capacity for disk in megabytes.")
+    public Integer getDisk() {
+        return disk;
+    }
+
+    public void setDisk(Integer disk) {
+        this.disk = disk;
+    }
+
+    @Pattern("[0-99]")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Description("Broker capacity for CPU utilization as a percentage (0 - 100).")
```

Comment:
isn't it set from 0 to 99 above?

Reply:
Updated to only allow a minimum integer value of 0 and a maximum integer value of 100